engine/lib/input.php

```diff
@@ -196,7 +194,7 @@ function elgg_is_empty($value) {
 function _elgg_htmlawed_tag_post_processor($element, $attributes = false) {
 	if ($attributes === false) {
 		// This is a closing tag. Prevent further processing to avoid inserting a duplicate tag
-		return "</${element}>";
+		return "</{$element}>";
```

Comment:
!

Reply:
should there be no tests?

Reply:
part of a callback of HTMLAwed not going to test this function specificly